PixaPicker/PixaSaveButtonDelegate.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  PixaSaveButtonDelegate.swift
+//  PixaPicker
+//
+//  Created by Thomas Caraway on 1/23/20.
+//  Copyright © 2020 Thomas Caraway. All rights reserved.
+//
+
+import Foundation
+
+protocol PixaSaveButtonDelegate{
```

Comment:
should be `protocol PixaSaveButtonDelegate: class {`  this guarentees whatever inherits the protocol is a Class. 

More reading on that here: https://stackoverflow.com/questions/31432449/retain-cycle-in-swift-delegate
Also more reading on Class vs Struct here: https://www.avanderlee.com/swift/struct-class-differences/